packages/odyssey-react-mui/src/AllowedProps.ts

```diff
@@ -10,11 +10,15 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-export type SeleniumProps = {
+export type AllowedProps = {
   /**
    * This prop puts a `data` attribute on an HTML element in this component with the value provided.
    *
    * @deprecated **WARNING:** You should be using Semantic Selectors instead of this property. This is a temporary measure for backwards compatibility with existing Selenium tests.
    */
   testId?: string;
+  /**
+   * This prop puts a `translate` attribute on an HTML element. It should be used to indicate whether text within the element should be translated.
+   */
+  translate?: "yes" | "no";
```

Comment:
Empty string too

```suggestion
  translate?: "" | "yes" | "no";
```

Reply:
Looks like React uses `translate?: 'yes' | 'no' | undefined;`. I could switch to matching this type but seems like empty string shouldn't actually be part of the typing?
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dc3f51b89bf7741b8d327311321ab6db041aa88d/types/react/index.d.ts#L1984